cypress/integration/join-page.spec.ts

```diff
@@ -130,20 +147,43 @@ describe('The join page', () => {
       });
 
       context('Entering bad values into the form', () => {
-        context('Entering a bad session id', () => {
+        describe('Using the malformed join code', () => {
+          for (const badJcId of [
+            'likeZoinksScoob',
+            '1',
+            '1234567',
+            // Eastern Arabic numerals, from Unicode.
+            '٢٢٠٩١٠',
```

Comment:
A few more fun tests cases:
```suggestion
            'likeZoinksScoob',
            '1',
            '-123456',
            '-12345',
            '1234567',
            '0123456',
            // Eastern Arabic numerals, from Unicode.
            '٢٢٠٩١٠',
            // Chinese/Japanese numerals
            '一二三四五六',
            // Fullwidth characters
            '１２３４５６'
```

Reply:
Nice, thanks :)